harness/src/main/java/org/jboss/pvt/harness/utils/ClassVersionInspector.java

```diff
@@ -101,4 +97,43 @@ public static boolean checkJarVersion (String filename, ClassVersion version) th
 
         return true;
     }
+
+    /**
+     * Support version scope
+     * @param filename
+     * @param minVersion
+     * @param maxVersion
+     * @return
+     * @throws PVTException
+     */
+    public static boolean checkJarVersion (String filename, ClassVersion minVersion, ClassVersion maxVersion) throws PVTException
+    {
```

Comment:
I have many files to commit, choosed wrong files for this PR, this file is for jdkcompatible validator.